packages/modules/src/modules/banners/guardianWeekly/GuardianWeeklyBanner.tsx

```diff
@@ -33,25 +33,36 @@ const closeComponentId = `${bannerId} : close`;
 const signInComponentId = `${bannerId} : sign in`;
 
 const mobileAndDesktopImg =
-    'https://i.guim.co.uk/img/media/670c3033e71f48296798480ac22a1123602e8466/0_0_500_240/500.png?quality=85&s=c1456ddb21a061ac1a199670637b5c5a';
+    'https://i.guim.co.uk/img/media/0682b069cf2e32b987ddcfbb2b549a93be61ae36/0_0_500_240/500.png?quality=85&s=1bf9bd8af343813bd6db3f009fccf385';
 
 const tabletImage =
+    'https://i.guim.co.uk/img/media/6d601169360b35c705412fbfa163c15f140efc2f/0_0_500_336/500.png?quality=85&s=21fb33ade343b7db222823c4d3160b7f';
+
+const mobileAndDesktopImgJan22 =
+    'https://i.guim.co.uk/img/media/670c3033e71f48296798480ac22a1123602e8466/0_0_500_240/500.png?quality=85&s=c1456ddb21a061ac1a199670637b5c5a';
+
+const tabletImageJan22 =
     'https://i.guim.co.uk/img/media/8d1fea851030b24328e05267fdb9bcd767248aeb/0_0_500_336/500.png?quality=85&s=102abc54f37796d6f668416bdbb993f1';
 
+const bannerMobileAndDesktopImg =
+    new Date() >= new Date('2022-01-24T08:00:00') ? mobileAndDesktopImgJan22 : mobileAndDesktopImg;
+const bannerTabletImageJan22 =
```

Comment:
For clarity I'd drop 'Jan22' from the end of this variable name, and name it `bannerTabletImage`.  Having 'Jan22' in the name implies the variable should only be used on/after Jan 22nd, this would bring it inline with the naming of `bannerMobileAndDesktopImg`.